src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -655,6 +654,20 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/admin/backup-task/delay", "POST", AuthorizationStatus.Operator)]
+        public async Task DelayBackupTask()
+        {
+            var id = GetLongQueryString("taskId");
+            var delay = GetTimeSpanQueryString("duration");
+            if (delay <= TimeSpan.Zero)
+                throw new ArgumentOutOfRangeException(nameof(delay));
+
+            var databaseName = GetStringQueryString("databaseName");
```

Comment:
can we use database here? That's the first place we have "databaseName" and url param. Let's be consistent.